core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/ChangeLog.java

```diff
@@ -7,42 +7,26 @@
 /**
  * A single entry in the database-generated change log for an object.
  */
-public class ChangeLog {
-  private String columnsChanged;
-  private String summary;
-  private Date time;
-  private User user;
-
-  public String getColumnsChanged() {
-    return columnsChanged;
-  }
-
-  public String getSummary() {
-    return summary;
-  }
-
-  public Date getTime() {
-    return time;
-  }
-
-  public void setColumnsChanged(String columnsChanged) {
-    this.columnsChanged = columnsChanged;
-  }
-
-  public void setSummary(String summary) {
-    this.summary = summary;
-  }
-
-  public void setTime(Date time) {
-    this.time = time;
-  }
-
-  public User getUser() {
-    return user;
-  }
-
-  public void setUser(User user) {
-    this.user = user;
-  }
+public interface ChangeLog {
+
+  public Long getId();
```

Comment:
Please Javadoc that the get/set ID methods returns/sets the associated entity's ID, not the changelog ID. Was this done to make the db delete query work?